Tasks/AndroidSigningV3/task.json

```diff
@@ -198,5 +198,10 @@
         "CouldNotFindVersionOfToolInAndroidHome": "Could not find %s version of %s inside ANDROID_HOME: %s",
         "GetLatestToolVersion": "Trying to get the latest installed version of tool: %s",
         "GetSpecifiedToolVersion": "Trying to get version %s of %s"
+    },
+    "restrictions": {
+        "commands": {
+            "mode": "restricted"
+        }
```

Comment:
Why no variable restrictions on this version?

Reply:
Thanks 👍 Just missed them for this task - added also